src/main/java/uk/gov/companieshouse/overseasentitiesapi/interceptor/UserAuthenticationInterceptor.java

```diff
@@ -74,4 +110,52 @@ public boolean preHandle(@NonNull HttpServletRequest request, @NonNull HttpServl
     protected Optional<TokenPermissions> getTokenPermissions(HttpServletRequest request) {
         return AuthorisationUtil.getTokenPermissions(request);
     }
+
+    protected boolean isRequestRoeUpdate(HttpServletRequest request) {
```

Comment:
Maybe this should be called hasOverseasEntityNumber() // or getAndValidateOverseasEntityNumber if we throw  an  error?

Reply:
we're really checking if the incoming request is for Update or Reg and branching off from there so I propose leaving it as is.

Reply:
I'm happy with this naming.